src/main/java/se/sundsvall/comfactfacade/api/model/Party.java

```diff
@@ -1,42 +1,89 @@
 package se.sundsvall.comfactfacade.api.model;
 
+import java.util.List;
+import java.util.Objects;
+
 import se.sundsvall.dept44.common.validators.annotation.ValidMobileNumber;
 import se.sundsvall.dept44.common.validators.annotation.ValidPersonalNumber;
 import se.sundsvall.dept44.common.validators.annotation.ValidUuid;
 
+import io.swagger.v3.oas.annotations.media.ArraySchema;
 import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
 
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@SuperBuilder(setterPrefix = "with")
 @Schema(description = "A party related to the signing process.")
-public record Party(
+public class Party {
```

Comment:
Is this class only used as a response? In that case, do we need validation?

Reply:
Its used both in the request and in the response